src/test/unit/spec1.js

```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664929/issues/source?bid=3233857&fileBranchId=3284028#l1)